src/NUnitFramework/framework/Internal/Filters/NotFilter.cs

```diff
@@ -57,15 +57,15 @@ public NotFilter( TestFilter baseFilter)
         /// <returns>True if the test passes the filter, otherwise false</returns>
         public override bool Pass(ITest test)
         {
-            return !BaseFilter.Pass(test);
-        }
+			return !BaseFilter.Match (test) && !BaseFilter.MatchParent (test);
+		}
 
-        /// <summary>
-        /// Check whether the filter matches a test
-        /// </summary>
-        /// <param name="test">The test to be matched</param>
-        /// <returns>True if it matches, otherwise false</returns>
-        public override bool Match( ITest test )
+		/// <summary>
+		/// Check whether the filter matches a test
+		/// </summary>
+		/// <param name="test">The test to be matched</param>
+		/// <returns>True if it matches, otherwise false</returns>
+		public override bool Match( ITest test )
```

Comment:
I thought they were part of the VS solution?  Isn't there a file somewhere that contains all the formatting rules for the solution.  I vaguely remember seeing it posted.